<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DotDesktopWizardDialog</class>
 <widget class="QDialog" name="DotDesktopWizardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>.desktop Wizard</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>.desktop File Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeComboBox">
       <property name="toolTip">
        <string>Select a target .desktop file format</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="dataTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basicTab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="typeEdit">
         <property name="toolTip">
          <string>Enter the type</string>
         </property>
         <property name="text">
          <string notr="true">Application</string>
         </property>
         <property name="placeholderText">
          <string>Mandatory</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Version:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="versionEdit">
         <property name="toolTip">
          <string>Enter the version of the .desktop specification</string>
         </property>
         <property name="text">
          <string notr="true">1.0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="nameEdit">
         <property name="toolTip">
          <string>Enter the application name</string>
         </property>
         <property name="placeholderText">
          <string>Mandatory</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Generic Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="genericNameEdit">
         <property name="toolTip">
          <string>Enter the generic application name</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="commentEdit">
         <property name="toolTip">
          <string>Enter a comment</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Icon:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="iconFileEdit">
         <property name="toolTip">
          <string>Enter the icon file name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Try Exec:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="tryExecEdit">
         <property name="toolTip">
          <string>Enter an executable command used to determine the application's presence</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Exec:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="execEdit">
         <property name="toolTip">
          <string>Enter the executable command</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="pathEdit">
         <property name="toolTip">
          <string>Enter the working directory</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Execute in Terminal:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="terminalCheckBox">
         <property name="toolTip">
          <string>Select to execute the application in a terminal</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>504</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Mimetype:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="mimetypeEdit">
         <property name="toolTip">
          <string>Enter the mime type</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Categories:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="categoriesEdit">
         <property name="toolTip">
          <string>Enter the list of categories separated by ';'</string>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QToolButton" name="categoriesButton">
         <property name="toolTip">
          <string>Press to select the categories via a dialog</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Actions:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="3">
        <widget class="E5ClearableLineEdit" name="actionsEdit">
         <property name="toolTip">
          <string>Enter the list of actions separated by ';'</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Only Show In:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="onlyShowEdit">
         <property name="toolTip">
          <string>Enter the list of environments to show in separated by ';'</string>
         </property>
         <property name="placeholderText">
          <string>Only one of 'Only Show In' or 'Not Show In' allowed</string>
         </property>
        </widget>
       </item>
       <item row="13" column="3">
        <widget class="QToolButton" name="onlyShowButton">
         <property name="toolTip">
          <string>Press to select the environments via a dialog</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Not Show In:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="notShowEdit">
         <property name="toolTip">
          <string>Enter the list of environments to not show in separated by ';'</string>
         </property>
         <property name="placeholderText">
          <string>Only one of 'Only Show In' or 'Not Show In' allowed</string>
         </property>
        </widget>
       </item>
       <item row="14" column="3">
        <widget class="QToolButton" name="notShowButton">
         <property name="toolTip">
          <string>Press to select the environments via a dialog</string>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="kdeTab">
      <attribute name="title">
       <string>KDE Plasma</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Encoding:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="kdeEncodingComboBox">
         <property name="toolTip">
          <string>Select the encoding of the file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>API:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="kdeApiComboBox">
         <property name="toolTip">
          <string>Select the Plasma API</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Service Type:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="kdeServiceTypeEdit">
         <property name="toolTip">
          <string>Enter the service type</string>
         </property>
         <property name="text">
          <string notr="true">Plasma/Applet</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Main Script:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="kdeMainScriptEdit">
         <property name="toolTip">
          <string>Enter the main script name</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Author:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="kdeAuthorEdit">
         <property name="toolTip">
          <string>Enter the author's name</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="kdeEmailEdit">
         <property name="toolTip">
          <string>Enter the auhor email</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="kdeNameEdit">
         <property name="toolTip">
          <string>Enter the application name</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Version:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="kdeVersionEdit">
         <property name="toolTip">
          <string>Enter the version number in the form 'major.minor[.patch[.sub]]'</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Website:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="kdeWebsiteEdit">
         <property name="toolTip">
          <string>Enter the URL of the website</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Category:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QComboBox" name="kdeCategoryComboBox">
         <property name="toolTip">
          <string>Select a category</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Depends:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="kdeDependsEdit">
         <property name="toolTip">
          <string>Enter the list of dependancies separated by ';'</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>License:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1" colspan="2">
        <widget class="E5ClearableLineEdit" name="kdeLicensEdit">
         <property name="toolTip">
          <string>Enter the license type</string>
         </property>
         <property name="text">
          <string notr="true">GPLv3</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Enabled by Default:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QCheckBox" name="kdeEnabledDefaultCheckBox">
         <property name="toolTip">
          <string>Select to enable this by default</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>508</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>37</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="unityTab">
      <attribute name="title">
       <string>Unity</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>X-Ayatana-Desktop-Shortcuts:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="E5ClearableLineEdit" name="unityShortcutsEdit">
         <property name="toolTip">
          <string>Enter the desktop shortcuts separated by ';'</string>
         </property>
         <property name="text">
          <string notr="true">Next;Previous</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>382</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="projectButton">
       <property name="toolTip">
        <string>Press to populate entry fields from project data</string>
       </property>
       <property name="text">
        <string>Populate from Project</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>E5Gui/E5LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>typeComboBox</tabstop>
  <tabstop>dataTabWidget</tabstop>
  <tabstop>typeEdit</tabstop>
  <tabstop>versionEdit</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>genericNameEdit</tabstop>
  <tabstop>commentEdit</tabstop>
  <tabstop>iconFileEdit</tabstop>
  <tabstop>tryExecEdit</tabstop>
  <tabstop>execEdit</tabstop>
  <tabstop>pathEdit</tabstop>
  <tabstop>terminalCheckBox</tabstop>
  <tabstop>mimetypeEdit</tabstop>
  <tabstop>categoriesEdit</tabstop>
  <tabstop>categoriesButton</tabstop>
  <tabstop>actionsEdit</tabstop>
  <tabstop>onlyShowEdit</tabstop>
  <tabstop>onlyShowButton</tabstop>
  <tabstop>notShowEdit</tabstop>
  <tabstop>notShowButton</tabstop>
  <tabstop>kdeEncodingComboBox</tabstop>
  <tabstop>kdeApiComboBox</tabstop>
  <tabstop>kdeServiceTypeEdit</tabstop>
  <tabstop>kdeMainScriptEdit</tabstop>
  <tabstop>kdeAuthorEdit</tabstop>
  <tabstop>kdeEmailEdit</tabstop>
  <tabstop>kdeNameEdit</tabstop>
  <tabstop>kdeVersionEdit</tabstop>
  <tabstop>kdeWebsiteEdit</tabstop>
  <tabstop>kdeCategoryComboBox</tabstop>
  <tabstop>kdeDependsEdit</tabstop>
  <tabstop>kdeLicensEdit</tabstop>
  <tabstop>kdeEnabledDefaultCheckBox</tabstop>
  <tabstop>unityShortcutsEdit</tabstop>
  <tabstop>projectButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DotDesktopWizardDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DotDesktopWizardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
